packages/labs/test-projects/test-element-a/package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Per convo with Justin, we're going to start putting private monorepo packages that won't ever be published in a `@lit-internal` "org", to make it clear what's internal and what's expected to come from the registry. We'll eventually do this for the `tests` package also.